src/main/java/mServer/crawler/sender/br/BrSendungDetailsTask.java

```diff
@@ -0,0 +1,93 @@
+
+  private final transient MediathekReader crawler;
+  private final ConcurrentLinkedQueue<String> filmIds;
+  private final transient Set<DatenFilm> convertedFilms;
+
+  public BrSendungDetailsTask(final MediathekReader aCrawler,
+      final ConcurrentLinkedQueue<String> aBrFilmIds) {
+    crawler = aCrawler;
+    filmIds = aBrFilmIds;
+    convertedFilms = ConcurrentHashMap.newKeySet();
+  }
+
+  
+  private ConcurrentLinkedQueue<String> createSubSet(
+      final ConcurrentLinkedQueue<String> aBaseQueue) {
+    final int halfSize = aBaseQueue.size() / 2;
+    final ConcurrentLinkedQueue<String> urlsToCrawlSubset = new ConcurrentLinkedQueue<>();
+    for (int i = 0; i < halfSize; i++) {
+      urlsToCrawlSubset.offer(aBaseQueue.poll());
+    }
+    return urlsToCrawlSubset;
+  }
+
+  private void filmIdsToFilms(final ConcurrentLinkedQueue<String> aFilmIds) {
+    for (final String filmId : aFilmIds) {
+      filmIdToFilm(filmId);
+    }
+    crawler.meldungProgress("");
+  }
+
+  private void filmIdToFilm(final String aFilmId) {
+    if (Config.getStop()) {
+      return;
+    }
+      
+    BrWebAccessHelper.handleWebAccessExecution(LOG, crawler, () -> {
+        
+        final Type optionalFilmType = new TypeToken<Optional<DatenFilm>>() {}.getType();
+        final Gson gson = new GsonBuilder()
+                .registerTypeAdapter(optionalFilmType, new BrFilmDeserializer(crawler, aFilmId)).create();
+ 
+        final String response = WebAccessHelper.getJsonResultFromPostAccess(new URL(Consts.BR_API_URL), String.format(QUERY_TEMPLATE, aFilmId));
+        
+        final Optional<DatenFilm> film = gson.fromJson(response, optionalFilmType);
+        if (film.isPresent()) {
+            convertedFilms.add(film.get());
+            //crawler.incrementAndGetActualCount();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)